src/main/kotlin/com/cognifide/gradle/aem/common/pkg/PackageOptions.kt

```diff
@@ -30,28 +30,41 @@ class PackageOptions(private val aem: AemExtension) : Serializable {
      * Custom path to Vault files that will be used to build CRX package.
      * Useful to share same files for all packages, like package thumbnail.
      */
-    val metaCommonDir = aem.obj.relativeDir(configDir, Package.META_PATH)
+    val metaCommonDir = aem.obj.relativeDir(commonDir, Package.META_PATH)
 
     /**
-     * Content path for OSGi bundle jars being placed in CRX package.
-     *
-     * Default convention assumes that sub-projects have separate bundle paths, because of potential re-installation of sub-packages.
-     * When all sub-projects will have same bundle path, reinstalling one sub-package may end with deletion of other bundles coming from another sub-package.
-     * As a consequence, generally it is recommended to avoid providing value by putting it into properties file.
-     *
-     * Beware that more nested bundle install directories are not supported by AEM by default (up to 4th depth level).
-     * That's the reason of using dots in sub-project names to avoid that limitation.
+     * Content path for AEM application placed in CRX package.
      */
-    val installPath = aem.obj.string {
+    val appPath = aem.obj.string {
         convention(aem.obj.provider {
             when (aem.project) {
-                aem.project.rootProject -> "/apps/${aem.project.rootProject.name}/install"
-                else -> "/apps/${aem.project.rootProject.name}/${aem.project.name}/install"
+                aem.project.rootProject -> "/apps/${aem.project.rootProject.name}"
+                else -> "/apps/${aem.project.rootProject.name}/${aem.project.name}"
             }
         })
-        aem.prop.string("package.installPath")?.let { set(it) }
+        aem.prop.string("package.appPath")?.let { set(it) }
```

Comment:
changing it via gradle.properties will automatically change also install path, install dir, config path and config dir which is very nice.